content/docs/user-guide/assets/asset-types.md

```diff
@@ -6,26 +6,46 @@ weight: 700
+| `.fontfamily`, `.font` | FontBuilderWorker | [LmbrCentral Gem](https://github.com/o3de/o3de/blob/development/Gems/LmbrCentral/Code/Source/Builders/CopyDependencyBuilder/FontBuilderWorker/FontBuilderWorker.h) | [Fonts](docs/user-guide/interactivity/user-interface/fonts) are used to render text. | Same as input, `.font` generates a `.font` product asset. |
+| `.tfx` | HairAssetBuilder | [AtomTressFX Gem](https://github.com/o3de/o3de/blob/development/Gems/AtomTressFX/Code/Builders/HairAssetBuilder.h) | [TressFX](docs/user-guide/gems/reference/rendering/amd/atom-tressfx) provides realistic hair and fur simulation. | `.tfxhair` |
+| `.lua` | Lua Worker Builder | [LmbrCentral Gem](https://github.com/o3de/o3de/blob/development/Gems/LmbrCentral/Code/Source/Builders/LuaBuilder/LuaBuilderWorker.h) | [Lua](docs/user-guide/scripting/lua) is a scripting language that can be used to author logic for your project. | `.luac` |
+| `.mock` | Mock Builder | [Automated Testing Project](https://github.com/o3de/o3de/blob/development/AutomatedTesting/Gem/PythonTests/PythonAssetBuilder/mock_asset_builder.py) | This builder is used to assist in some automated testing, run via the AutomatedTesting project. It's a useful example for teams that wish to author builders for custom asset types in their own project, in Python. | `.mock_asset` |
+| `.motionset` | MotionSetBuilderWorker | [EmotionFX Gem](https://github.com/o3de/o3de/blob/development/Gems/EMotionFX/Code/EMotionFX/Pipeline/EMotionFXBuilder/MotionSetBuilderWorker.h) | A [motion set](docs/user-guide/visualization/animation/character-editor/concepts-and-terms) contains a list of motions. | `.motionset` |
+| `.pass` | Pass Asset Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/RPI/Code/Source/RPI.Builders/Pass/PassBuilder.h) | [Passes](docs/atom-guide/dev-guide/passes) are logical groupings of render work, with a defined input and output. | `.pass` |
+| `.precompiledshader` | Precompiled Shader Builder | [Atom Gem](https://github.com/o3de/o3de/blob/development/Gems/Atom/Asset/Shader/Code/Source/Editor/AzslShaderBuilderSystemComponent.h) | - | `.azshader` |
+| `.prefab` | Prefab Builder | [Prefab Gem](https://github.com/o3de/o3de/blob/development/Gems/Prefab/PrefabBuilder/PrefabBuilderComponent.h) | [Prefabs](docs/user-guide/interactivity/prefabs) are a foundational element for building an O3DE project, a prefab is a collection of entities. | `.spawnable` |
+| `.ts` | Qt Translation File Builder | [LmbrCentral Gem](https://github.com/o3de/o3de/blob/development/Gems/LmbrCentral/Code/Source/Builders/TranslationBuilder/TranslationBuilderComponent.h) | Qt translation files. Qt is only enabled in tools, so these can be useful for translating tools to different languages, but not projects. | `.qm` |
```

Comment:
We really need to clean up some terminology. There are non-Editor tools, like Asset Processor and Asset Bundler, that can use the translation files. I've changed it to your suggestion for now because I think we need to do an entire-documentation pass on the terminology.

Reply:
I'll pass this feedback on to the rest of the team.